packages/tasit-sdk/src/tasitSdk.test.js

```diff
@@ -0,0 +1,9 @@
+import { Account } from "./index";
```

Comment:
For some reason my gut reaction is that I would have done
`import { Account } from "./tasitSdk";`
even though the current approach is actually more directly testing what the user would actually be importing, which makes for an arguably better test. @marcelomorgado what do you think?

Reply:
I agree with you, I've fixed it on PR https://github.com/tasitlabs/TasitSDK/blob/c1df5e13930835f64c8c5e37a91d2a4b5e58836e/packages/tasit-sdk/src/tasitSdk.test.js#L1

Reply:
I can fix it on this branch either. You call.

Reply:
It being changed in the other PR is fine with me! Thanks.